app/components/CameraComponent.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import {
+  Alert,
+  Dimensions,
+  Image,
+  StyleSheet,
+  View,
+} from 'react-native';
+import Camera from 'react-native-camera';
+import Button from 'react-native-button';
+
+export default class CameraComponent extends Component {
+
+  state = {
+    image: null,
+  };
+
+  _takePicture() {
```

Comment:
If you define this as an arrow function, you don't need to call `.bind()` on it before passing it into an `onPress`.

i.e. change it to
```
_takePicture = () => {
  // code here
};
```